### DISCUSSION TITLE
2023-11-09 CUE Modules Feedback Session / office hours

### DISCUSSION THREAD

**Discussion by myitcv** [cue-lang/cue#2655] [Community Calls]
This discussion is a placeholder for the next instalment in the Modules Feedback Sessions (office hours). 

See this discussion for context on the modules proposal: https://github.com/cue-lang/cue/discussions/2330

The next session will be on Thursday, **9 Nov, 2023 at 1600 UTC** (note the clocks will have changed in the UK, hence the "different" time).

Please reply with any questions/topics you would like to see covered.

### Rough agenda

* Discussion of recent progress/decisions
* Auth update
* Exploration of `cuelabs.dev/go/oci/ociregistry` module
* Demo of routing modules from different OCI registries
* Demo of upload to a non-central registry
* Walk through of end-to-end test setup

### Meeting details

https://meet.google.com/wis-hrtw-isf

This session will also be recorded for those unable to join us live.

### CUE Community shared calendar

[Subscribe to the shared calendar](https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).  

### Recording

https://youtu.be/StIsK_Ruajw

### Slides/materials

https://docs.google.com/presentation/d/14RWXNhOTg-LyWVhLUpZh_P7Edo5Ovg154xNfnNB__SY/edit?usp=sharing

### Auto-generated transcript 

https://docs.google.com/document/d/1dVYBvldSnUz7x1nYM3eZNWHaOixNkSYxyg48wMoPDvQ/edit?usp=sharing

**Comment by dominikdm** [comment]
Updated with recording, slides and transcript.
